MCTSampleApp/MCTSampleApp.xcodeproj/project.pbxproj

```diff
@@ -367,9 +367,9 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 3;
+				CURRENT_PROJECT_VERSION = 0;
```

Comment:
Is this intentional (resetting the project version to 0)?

Reply:
This is the build. The project version is the marketing version (3.0) and the build (0) which then are combined for a semantic version of 3.0.0